galaxy.xsd

```diff
@@ -2099,13 +2115,12 @@ code file wraps a metadata collection to return metadata parameters wrapped
 according to the Metadata spec.
 
 ```xml
-<command>
```

Comment:
I didn't notice that was already inside a CDATA, but it's possible: https://en.wikipedia.org/wiki/CDATA#Nesting